plugin-api/src/main/java/org/hyperledger/besu/plugin/data/TransactionType.java

```diff
@@ -21,10 +21,11 @@
 public enum TransactionType {
   FRONTIER(0xf8 /* this is serialized as 0x0 in TransactionCompleteResult */),
   ACCESS_LIST(0x01),
-  EIP1559(0x02);
+  EIP1559(0x02),
+  EIP4844(0x03);
 
   private static final Set<TransactionType> ACCESS_LIST_SUPPORTED_TRANSACTION_TYPES =
-      Set.of(ACCESS_LIST, EIP1559);
+      Set.of(ACCESS_LIST, EIP1559, EIP4844);
```

Comment:
Please sync with @gezero that in his PR also set this with a different name https://github.com/hyperledger/besu/pull/4919/files#diff-f4664e850b91579cf65137f12d71d2269b6dc4d3f1c59ee5f65ae0782947809a